patches/minecraft/net/minecraft/world/storage/loot/LootTable.java.patch

```diff
@@ -1,6 +1,14 @@
 --- a/net/minecraft/world/storage/loot/LootTable.java
 +++ b/net/minecraft/world/storage/loot/LootTable.java
-@@ -33,13 +33,13 @@
+@@ -24,6 +24,7 @@
+ import net.minecraft.util.math.MathHelper;
+ import net.minecraft.world.storage.loot.functions.ILootFunction;
+ import net.minecraft.world.storage.loot.functions.LootFunctionManager;
++
```

Comment:
where did this come from

Reply:
I'll double-check that. I thought I'd been careful not to import anything, but its possible I did on accident.

Reply:
Ah, the blank line. Fixed that (local for the moment), not sure how it snuck in.